backend/matching-service/index.js

```diff
@@ -33,4 +33,6 @@ require('./routes/matching.routes')(app);
 
 app.listen(PORT, () => {
   console.log(`matching-service listening on port ${PORT}`);
-});
\ No newline at end of file
+});
+
+module.exports = app;
```

Comment:
Is this necessary?

Reply:
This is required to call the server in the unit tests.